extensions.json

```
@@ -283,6 +283,21 @@
       "id": "zhuangtongfa.material-theme",
       "version": "3.2.4",
       "repository": "https://github.com/Binaryify/OneDark-Pro"
+    },
+    {
+      "id": "ms-vscode.node-debug",
+      "version": "1.35.3",
```

Why not the latest available version? https://github.com/microsoft/vscode-node-debug/releases

```suggestion
      "version": "1.41.1",
```

---

See https://github.com/open-vsx/publish-extensions/pull/16#issuecomment-603300080